pyproject.toml

```diff
@@ -13,6 +13,7 @@ etl-wizard = 'apps.wizard.cli:cli'
 fasttrack = 'apps.fasttrack.cli:cli'
 compare = 'etl.compare:cli'
 backport = 'apps.backport.backport:backport_cli'
+etl-step-update = 'apps.step_update.cli:cli'
```

Comment:
We are moving away from using `etl-*` like commands. Instead, we should put these under our single client `etl`.

More on the motivation for this here: https://github.com/owid/etl/pull/2293